frontend/src/metabase/query_builder/components/QueryModals.jsx

```diff
@@ -181,6 +182,24 @@ export default class QueryModals extends React.Component {
       <Modal full onClose={onCloseModal}>
         <QuestionEmbedWidget card={this.props.card} onClose={onCloseModal} />
       </Modal>
+    ) : modal === "clone" ? (
+      <Modal onClose={onCloseModal}>
+        <EntityCopyModal
+          entityType="questions"
+          entityObject={this.props.card}
+          copy={async formValues => {
+            const object = await this.props.onCreate({
+              ...this.props.card,
+              ...formValues,
+              description: formValues.description || null,
+              id: undefined,
```

Comment:
I didn't test, but I don't think you need this line. `EntityCopyModal` passes `dissoc(entityObject, "id")`

Reply:
You're right -- removed `id: undefined`